src/js/components/CheckBox/doc.js

```diff
@@ -21,6 +21,14 @@ export const doc = (CheckBox) => {
     checked: PropTypes.bool
       .description('Same as React <input checked={} />')
       .defaultValue(false),
+    children: PropTypes.func.description(
```

Comment:
This change will now need to be made in https://github.com/grommet/grommet-site/blob/master/src/screens/CheckBox.js